package.json

```diff
@@ -32,8 +32,7 @@
   "devDependencies": {
     "eslint": "6.8.0",
     "eslint-config-airbnb-base": "14.1.0",
-    "eslint-plugin-import": "2.20.2",
-    "karma": "4.4.1"
```

Comment:
Removed `karma` from `devDependencies` because it's not being used in development.

Reply:
That's only because I'm a slacker and never wrote tests for this repo. :-)

We have `karma` listed as a `peerDependency`.  I believe the practice is to list an exact version in `devDependencies` for anything listed in `peerDependencies`.

I agree it's a little weird since there are no tests, but *if* there were, wouldn't we want this entry?

Reply:
I suppose. I'll add it back.

Reply:
I like your confidence in my suggestion.